Kill warnings in Armadillo detection code to make compiler output empty
--- gdal-1.11.0/configure.in.orig	2014-06-14 15:45:50.005039893 +0200
+++ gdal-1.11.0/configure.in	2014-06-14 16:46:47.261630331 +0200
@@ -4227,7 +4227,7 @@
     rm -f testarmadillo.*
     rm -f testarmadillo
     echo '#include <armadillo>' > testarmadillo.cpp
-    echo 'int main(int argc, char** argv) { arma::mat matInput(2,2); const arma::mat& matInv = arma::inv(matInput); return 0; } ' >> testarmadillo.cpp
+    echo 'int main(int, char**) { arma::mat matInput(2,2); const arma::mat& matInv = arma::inv(matInput); (void)matInv; return 0; } ' >> testarmadillo.cpp
     if test -z "`${CXX} ${CPPFLAGS} testarmadillo.cpp -o testarmadillo -larmadillo 2>&1`" ; then
         HAVE_ARMADILLO=yes
         LIBS="-larmadillo ${LIBS}"
